test/test_torch.py

```diff
@@ -5317,8 +5317,18 @@ def test_isfinite_int(self):
         self.assertEqual(torch.isfinite(x), torch.ByteTensor([1, 1, 1]))
 
     def test_isinf(self):
-        x = torch.Tensor([1, inf, 2, -inf, nan])
-        self.assertEqual(torch.isinf(x), torch.ByteTensor([0, 1, 0, 1, 0]))
+        t1 = torch.Tensor([1, inf, 2, -inf, nan])
```

Comment:
Could you add a test to test/test_cuda.py as well? It should reference the test torch mixin: https://github.com/pytorch/pytorch/blob/3da4a0473368e9ac18376a9797c46c28cf0bb45d/test/test_cuda.py#L938